modules/apps/site-initializer/site-initializer-testray/extra/java-function/src/main/java/com/liferay/site/initializer/testray/extra/java/function/ImportResults.java

```diff
@@ -544,5 +832,8 @@ private void _unTarGzip(byte[] bytes) throws Exception {
 	private final DocumentBuilder _documentBuilder;
 	private final DocumentBuilderFactory _documentBuilderFactory;
 	private final Storage _storage;
+	private long componentId;
```

Comment:
Why is attribute?
Can we put inside method as variables and pass to another methods as parameters?